code/components/citizen-server-impl/src/state/ServerGameState_Scripting.cpp

```diff
@@ -718,7 +724,12 @@ static void Init()
 
 		if (!wheelsFine && context.GetArgumentCount() > 1)
```

Comment:
Are the argument count checks here required? Usually this is only done for natives that return pointers.

Reply:
Yeah, no clue why this was added here initially. Seems like those count checks are included in a lot of native definitions while not being required. Will remove here as we're already changing the implementation👍🏼